crescendo -official robot code/src/main/java/frc/robot/utils/Vortex.java

```diff
@@ -0,0 +1,5 @@
+package frc.robot.utils;
+
+public class Vortex {
```

Comment:
what are the differences in api between neo and vortex ?

Reply:
Spark max and spark flex sadly

Reply:
Problem is most, and isn't it abstract?